src/v3/test/integration/byol.test.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import mockResponse from '@okta/mocks/data/idp/idx/authenticator-enroll-data-phone.json';
+import loginFooBundle from '@okta/mocks/labels/json/login_foo.json';
+import countryFooBundle from '@okta/mocks/labels/json/country_foo.json';
+import { waitFor, cleanup } from '@testing-library/preact';
+import { rest } from 'msw';
+import { setupServer, SetupServerApi } from 'msw/node';
+import { setup } from './util';
+
+describe('byol', () => {
+  beforeEach(() => {
+    // language is cached after first test so much disabling caching for this test
```

Comment:
> so much

typo?

Reply:
Should have read `language is cached after first test so disabling caching for this test` 😅